pkg/kafka/consumer/config.go

```diff
@@ -63,5 +64,10 @@ func (c *Configuration) NewConsumer(logger *zap.Logger) (Consumer, error) {
 	if err := c.AuthenticationConfig.SetConfiguration(&saramaConfig.Config, logger); err != nil {
 		return nil, err
 	}
+	// cluster.NewConfig() uses sarama.NewConfig() to create the config.
+	// However the Jaeger OTEL module pulls in newer samara version (from OTEL collector)
+	// that does not set saramaConfig.Consumer.Offsets.CommitInterval to its default value 1s.
+	// then the samara-cluster fails if the default interval is not 1s.
+	saramaConfig.Consumer.Offsets.CommitInterval = time.Second
```

Comment:
Should this check whether the value has already been set, for use in non-OTEL context? Or would it only ever be the default value?

Reply:
One second is always the default value we never override it.

Reply:
Note that the whole kafka implementation will be replaced to OTEL one that does not use samara-cluster https://github.com/jaegertracing/jaeger/pull/2494